Content.Client/GameObjects/Components/Observer/AcceptCloningWindow.cs

```diff
@@ -25,7 +25,7 @@ public AcceptCloningWindow()
                         {
                             (new Label()
                             {
-                                Text = Loc.GetString("You are being cloned!\nTransfer your soul to the clone body?")
+                                Text = Loc.GetString("accept-cloning-window-prompt-text-part-1" + "\n" + "accept-cloning-window-prompt-text-part-2")
```

Comment:
This doesn't work. There's no Id called `"accept-cloning-window-prompt-text-part-1\naccept-cloning-window-prompt-text-part-2"` which is technically what you're passing in now.

Also I'd recommend this be one message, with the newline actually part of the message.